fabric-renderer-indigo/src/main/java/net/fabricmc/indigo/Indigo.java

```diff
@@ -23,6 +23,8 @@
 import org.apache.logging.log4j.Logger;
 
 public class Indigo implements ClientModInitializer {
+	// TODO: Add configuration option/system property/etc.
+	public static final boolean ALWAYS_TESSELATE_INDIGO = false;
```

Comment:
Default needs to be true or there needs to be a very obvious and easy (player-visible) way to change it.  

Default = false means all lighting improvements (aesthetic and performance) will also be disabled for the vast majority of blocks rendered.  That is different from current behavior and probably not desired/expected by most players.

Reply:
Yes, there's a TODO here for that reason